src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -5421,6 +5422,57 @@ class MyProtoGapicClient
         )->wait();
     }
 
+    /**
+     * Define a service with a reserved name
+     *
+     * Sample code:
+     * ```
+     * $myProtoClient = new MyProtoClient();
+     * try {
+     *     $response = $myProtoClient->getNamespace();
+     * } finally {
+     *     $myProtoClient->close();
+     * }
+     * ```
+     *
+     * @param array $optionalArgs {
+     *     Optional.
+     *     @type SubMessage[] $mylist
+     *     @type SubMessage $myfield
+     *     @type SubMessage $secondfield
+     *     @type RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     * }
+     *
+     * @return \Google\Example\Library\V1\Namespace
```

Comment:
It looks like this should be `PBNamespace`?

Reply:
Good catch! This is now fixed